src/components/Api.js

```diff
@@ -0,0 +1,80 @@
+export default class Api {
+  constructor(options) {
+    this._url = options.url;
+    this._headers = options.headers;
+  }
+
+  _getRes(res) {
+    return res.ok ? res.json() : Promise.reject(`Error: ${res.status}`);
+  }
+
+  getProfileApi() {
+    return fetch(this._url + "/users/me", {
```

Comment:
You can use the templated literals here

Reply:
is this a mistake that needs to be fixed? Also raises a question, is it better to use Template strings?

Reply:
This "Could be better". It's a clearer and more modern approach